src/components/accounts/auto_import/AutoImport.js

```diff
@@ -3,44 +3,49 @@ import { Translate } from 'react-localize-redux';
 import { useDispatch, useSelector } from 'react-redux';
 import styled from 'styled-components';
 
-import { recoverAccountSeedPhrase, refreshAccount, redirectTo } from '../../../actions/account';
+import {
+    recoverAccountSecretKey,
+    refreshAccount,
+    redirectTo,
+    clearAccountState
+} from '../../../actions/account';
 import { Mixpanel } from '../../../mixpanel/index';
 import FormButton from '../../common/FormButton';
 import Container from '../../common/styled/Container.css';
 
 const StyledContainer = styled(Container)`
-    h1 {
-        margin-top: 50px;
-    }
-
-    button {
-        margin: 40px auto 0 auto !important;
-        display: block !important;
-        width: 100% !important;
+    &&& {
+        h1 {
+            margin-top: 50px;
+            text-align: center;
+        }
+    
+        button {
+            margin: 40px auto 0 auto;
+            display: block;
+            width: 100%;
+        }
     }
 `;
 
-export function AutoImport() {
+export function AutoImport({ secretKey, accountId, importType }) {
     const dispatch = useDispatch();
     const { location } = useSelector(({ router }) => router);
     const [recovering, setRecovering] = useState(false);
-    const string = decodeURI(window.location.hash.substring(1));
-    const hasAccountId = string.includes('/');
-    const accountId = hasAccountId ? string.split('/')[0] : null;
-    const seedPhrase = hasAccountId ? string.split('/')[1] : string;
     const successUrl = location.query.success_url;
     const failureUrl = location.query.failure_url;
     
     useEffect(() => {
-        recoverWithSeedPhrase();
+        recoverWithSecretKey();
     }, []);
 
-    const recoverWithSeedPhrase = async () => {
-        await Mixpanel.withTracking("IE-SP Recovery with seed phrase auto",
+    const recoverWithSecretKey = async () => {
+        await Mixpanel.withTracking(`IE-SP Recovery with ${importType} auto`,
             async () => {
                 setRecovering(true);
-                await dispatch(recoverAccountSeedPhrase(seedPhrase, accountId));
+                await dispatch(recoverAccountSecretKey(secretKey, accountId, false));
```

Comment:
Although this argument name makes it _seem_ like we should pass in `false` here, the fact that previously we were not passing in any argument and that `recoverAccountSeedPhrase()` defaults this value to `true` makes this a change in behaviour compared to our existing code.  I dug into the implementation of this function and noted that what is labelled `fromSeedPhraseRecovery` actually just means `shouldCreateFullAccessKey` as far as I can tell -- which we want to be _true_ for this code, still, :)